File: api/Entities/Admin/Order/JobDescription.cs
using System.ComponentModel.DataAnnotations;

namespace api.Entities.Admin.Order
{
    public class JobDescription: BaseEntity
    {
        public int OrderItemId { get; set; }    //foreign key
        [Required, MaxLength(250)]
        public string JobDescInBrief { get; set; }
        public string QualificationDesired { get; set; }
        [Range(0,40)]
        public int ExpDesiredMin { get; set; }
        [Range(0,40)]
        public int ExpDesiredMax { get; set; }
        [Range(18,80)]
        public int MinAge { get; set; }
        [Range(18,70)]
        public int MaxAge { get; set; }
        
    }
}

File: client/src/app/_models/admin/jobDescription.ts
export interface IJobDescription
{
     id: number; 
     orderItemId: number; 
     //orderId: number; 
     orderNo: number; 
     jobDescInBrief: string; 
     qualificationDesired: string;
     expDesiredMin: number; 
     expDesiredMax: number; 
     minAge: number; 
     maxAge: number;
}